code/game/objects/effects/decals/wrestling.dm

```diff
@@ -0,0 +1,76 @@
+/obj/decal/wrestling/rope
+	name = "Wrestling Ropes"
+	desc = "Do not exit the ring."
+	density = 1
```

Comment:
Pretty sure Density and Anchored should be TRUE and FALSE instead of 0 and 1. Goes for a lot of places in this PR